packages/bot/src/helpers.ts

```diff
@@ -109,14 +121,14 @@ export function createBotHelpers(bot: Bot): BotHelpers {
     createForumThread: async (channelId, options, reason) => {
       return bot.transformers.channel(bot, { channel: snakelize(await bot.rest.createForumThread(channelId, options, reason)) })
     },
-    createGlobalApplicationCommand: async (command) => {
-      return bot.transformers.applicationCommand(bot, snakelize(await bot.rest.createGlobalApplicationCommand(command)))
+    createGlobalApplicationCommand: async (command, bearer) => {
```

Comment:
How would you feel about making this bearer into an options?

```ts
interface XXXX {
  bearerToken: string;
}
```

The reasoning/motivation for this is to make it so that for the future any changes would be non-breaking. For example, if tomorrow discord adds another option for this function, it would make this a breaking change, since u may end up having `createGlobalApplicationCommand(command, undefined, xyz)`

I would prefer we start making new things using objects to prevent breaking changes in future.

Reply:
I have made the change, let me know if i should also do this for all the other methods that accept a bearer token